interface/clickmap/template/general_new.html

```diff
@@ -11,60 +11,54 @@
 	</head>
```

Comment:
now that jquery-ui is removed, should be able to drop line 6 and change line 4 to:
```
{headerTemplate}
```